src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx

```diff
@@ -399,6 +398,27 @@ Here are some additional distributed tracing UI details, rules, and limits:
     * **UI display limits.** Orphaned spans may result if a trace exceeds the 10K span display limit.
   </Collapser>
 
+  <Collapser
+    id="preserved-traces"
+    title="Preserved traces"
+  >
+    A preserved distributed trace is like an archived snapshot of the original trace that has been previously viewed and has surpassed the retention period. By default, unless you have purchased extended retention, a full trace is onlyavailable for 7 days. Preserved traces can exist for up to 1 year, and generally function like the original trace.
```

Comment:
I like framing this kind of trace as an archived snapshot, but we can unspool this first sentence for clarity. Maybe something like 'Preserved traces are similar to archived snapshots. They archive a trace that has been previously viewed and has surpassed the retention period.'